test/unit/lifecycle/properties.js

```diff
@@ -25,7 +26,11 @@ describe('lifecycle/property', () => {
   });
 
   describe('props declared as attributes with ES2015 classes are linked', () => {
+    const skip = !classStaticsInheritance();
```

Comment:
Is that relevant? `setPrototypeOf` won't exist in older browsers, so it's just a simpler way of checking.